src/StorageSync/StorageSync/help/Get-AzStorageSyncServer.md

```diff
@@ -48,9 +48,9 @@ This command gets all servers registered to a specific storage sync service.
 The credentials, account, tenant, and subscription used for communication with Azure.
 
 ```yaml
-Type: Microsoft.Azure.Commands.Common.Authentication.Abstractions.Core.IAzureContextContainer
+Type: IAzureContextContainer
```

Comment:
It looks like you didn't generate help with the `-UseFullTypeName` parameter as specified here: https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md

This will cause links to .Net SDK documentation to fail on docs.microsoft.com, please update the help using the suggested parameters

Reply:
Indeed, that parameter wasn't in use. Added and regenerated.